gem/lib/earlgrey/cli.rb

```diff
@@ -19,24 +19,26 @@ class CLI < Thor
     package_name 'EarlGrey'
 
     desc 'install', 'Installs EarlGrey into an Xcode unit test target'
+    method_option :carthage, type: :boolean, default: true
     method_option :project, aliases: '-p', type: :string, required: false, desc: 'Project'
-    method_option :target,  aliases: '-t', type: :string, required: true, desc: 'EarlGreyTestTarget'
     method_option :scheme,  aliases: '-s', type: :string, required: false, desc: 'EarlGreyTestTarget.xcscheme'
-    method_option :swift,    type: :boolean, default: true
-    method_option :carthage, type: :boolean, default: true
+    method_option :swift, type: :boolean, default: true
+    method_option :swift_version, type: :string, default: '3.0', enum: ['2.2', '3.0']
```

Comment:
Oh, that's just a lint check. Use `%w[2.2 3.0]` instead.

In Ruby the token after the w can be anything.

```
%w*a b*
=> ["a", "b"]
```